composer.json

```diff
@@ -16,10 +16,6 @@
         "install-phpcs": "mkdir -p phpcs && cd phpcs && COMPOSER_BIN_DIR=../vendor/bin composer require squizlabs/php_codesniffer:^2.7",
```

Comment:
Never version of php_codesniffer available?

Reply:
Fixed, thanks